tests/uniswap/main.rs

```diff
@@ -29,5 +30,17 @@ fn uniswap_clusters() {
         final_state.extend(state);
         final_txs.extend(txs);
     }
-    common::test_execute_revm(InMemoryStorage::new(final_state, []), final_txs)
+
+    let mut bytecodes: AHashMap<B256, EvmCode> = AHashMap::new();
```

Comment:
Let's return `bytecodes` from `generate_cluster`.

Reply:
resolved in the latest commit